src/picongpu/submit/titan-ornl/batch_profile.tpl

```diff
@@ -38,7 +36,7 @@ TBG_nodes=!TBG_tasks
 #PBS -N !TBG_jobName
 #PBS -l nodes=!TBG_nodes
 # send me a mail on (b)egin, (e)nd, (a)bortion
```

Comment:
yes that works.
or just skipping the "me" because the mail is written the line below